days/day-05/solutions/day05.stektpotet.py

```diff
@@ -0,0 +1,6 @@
+import sys
+ids = {int(''.join('0' if c == 'F' or c == 'L' else '1' for c in k), 2) for k in sys.stdin.read().split('\n')[:-1]}
+print(max(ids))
+for i in range(min(ids), max(ids)):
+    if not i in ids:
+        print(i)
```

Comment:
Wow! This is elegant.  Does not require a separate intermediate storage, like many other solutions here. 2x approved! :tada:

Reply:
## Now we're thinking with 🐍 
Sidenote:
Turns out I actually did a minor mixup here, though python says **I'll allow it!**,  the preferred way is: `if i not in ids`
Runs just the same either way though 😅

Reply:
Wow. Python MVP! :pray: